scene/resources/packed_scene.h

```diff
@@ -59,6 +60,9 @@ class SceneState : public RefCounted {
 		int name = 0;
 		int instance = 0;
 		int index = 0;
+		int32_t local_id = 0;
+		int32_t parent_local_id = 0;
+		int32_t local_parent_owner_id = 0;
```

Comment:
Why does it use `int32_t` whereas everything around it uses an `int`?

Reply:
I think I had a conversation about this either on rocketchat or discord and int32 is more explicit (and it's safer to have this large precision to cover a large potential node count); int can vary. So I was told that it's fine to be explicit.